### PR TITLE
Add basic sites view (no CSS).

### PR DESCRIPTION
**Pull request by LyonesGamer**
fixes issue #18 

## Changes in this PR.

- [X] Create base template
- [X] Create statics, including JQuery.js, CSS stylesheet, and favicon
- [X] Create sites template with:
  - [X] Search bar
  - [X] Google maps view with markers
  - [X] Dropdown list of sites
- [X] Redirection to site pages
- [X] Different colored markers on map
- [ ] Actual data lookup (currently uses test data)

## Testing this PR.

1. Run the docker container (``docker-compose up``)
2. Go to ``http://localhost:8000/streamwebs/sites``
3. Type in search bar, click on map markers, and select sites from dropdown

### Expected Output.

Upon clicking a site in the search bar, clicking the link in an info window on the map, or submitting the selection from the dropdown, the page should print to console ``Going to site ${slug}`` where ``slug`` is the slug of the site the user selected, and the page should redirect to ``/site/${slug}``, which should print ``Viewing site for page ${slug}``.

@osuosl/devs

### CODE REVIEW

**Comment by LyonesGamer** [comment]
Note that although GitHub claims this PR is 10,000 lines added, almost all are from JQuery.js, which I elected to serve locally instead of depending on a third-party server.

**Comment by ElijahCaine** [comment]
@LyonesGamer Why not use something like Bower to add the Jquery dependency on install? Or use their CDN?

**Comment by Kennric** [comment]
@LyonesGamer - as @ElijahCaine says above, we typically use Bower for javascript assets, can you add a bower file and remove the actual jquery libraries?

**Comment by LyonesGamer** [comment]
@ElijahCaine @Kennric The last PR moved to use a combination of Bower and Pipeline for dependencies (currently just JQuery).

**Comment by Kennric** [comment]
Looks like an ok base view. +1

File: streamwebs_frontend/streamwebs/templates/streamwebs/sites.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
+{% extends "streamwebs/base.html" %}
+
+{% load staticfiles %}
+
+{% block title %}Sites - Streamwebs{% endblock %}
+
+{% block scripts %}
+<script src="http://maps.google.com/maps/api/js"></script>
+<script type="application/javascript">
```
Comment (ElijahCaine) [inline streamwebs_frontend/streamwebs/templates/streamwebs/sites.html]: In-line javascript?

Comment (LyonesGamer) [inline streamwebs_frontend/streamwebs/templates/streamwebs/sites.html]: For a small, loose collection of page-specific functions. I can move it out into a separate file, but given that the file would pretty much only be used by this page, I'm not sure I see the point.

Comment (ElijahCaine) [inline streamwebs_frontend/streamwebs/templates/streamwebs/sites.html]: You should factor it out to it's own file. Really all of our custom JS should be in it's own file so the user can cache it on their end and we don't need to worry about looking through all of our templates to edit dynamic functionality.

**Comment by Kennric** [comment]
I think this is a good start. +1

**Comment by ElijahCaine** [comment]
@LyonesGamer When I go to localhost:8000/streamwebs/sites I just get a blank screen. This is the output in the terminal:
```
$ sudo docker-compose up
Starting streamwebs_postgres_host_1
Starting streamwebs_web_1
Attaching to streamwebs_postgres_host_1, streamwebs_web_1
postgres_host_1  | LOG:  database system was interrupted; last known up at 2016-06-13 21:54:41 UTC
postgres_host_1  | LOG:  database system was not properly shut down; automatic recovery in progress
postgres_host_1  | LOG:  invalid record length at 0/2EB1540
postgres_host_1  | LOG:  redo is not required
postgres_host_1  | LOG:  MultiXact member wraparound protections are now enabled
postgres_host_1  | LOG:  database system is ready to accept connections
postgres_host_1  | LOG:  autovacuum launcher started
web_1            | [13/Jun/2016 14:55:33] "GET /streamwebs/sites HTTP/1.1" 200 3366
web_1            | [13/Jun/2016 14:55:33] "GET /static/streamwebs/style.css HTTP/1.1" 304 0
web_1            | [13/Jun/2016 14:55:33] "GET /static/streamwebs/js/sites.js HTTP/1.1" 304 0
```
Inspecting the web page the head loads but nothing in the body (like the search bar or map to click on).

**Comment by LyonesGamer** [comment]
@ElijahCaine Was the fault of an unclosed script tag. Should be fixed now. We should really also add some simple HTML and JS linting in the Travis tests...

**Comment by ElijahCaine** [comment]
+1 after the docker changes

**Comment by Kennric** [comment]
+1

**Pull request merged**